libs/templates/src/lib/trade/app-download/data.tsx

```diff
@@ -0,0 +1,77 @@
+import { OptimizedImage } from '@deriv-com/components';
+import {
+  StandaloneGlobeRegularIcon,
+  LabelPairedWindowsIcon,
+  StandaloneIosIcon,
+  StandaloneLinuxIcon,
+  StandaloneHuaweiAppGalleryIcon,
+  StandaloneGooglePlayIcon,
+  StandaloneAppStoreIosIcon,
+} from '@deriv/quill-icons';
+
+export const data = {
+  content: (
+    <div className="flex">
+      <OptimizedImage
+        className="hidden lg:block"
+        imageName="trade/features/hk_trade_feature_desktop_mt5.png"
+        width={608}
+        height={536}
+        alt="MT5 App"
+      />
+      <OptimizedImage
+        className="block lg:hidden"
+        imageName="trade/features/hk_trade_feature_mobile_mt5.png"
+        width={809}
+        height={460}
+        alt="MT5 App"
+      />
```

Comment:
`OptimizedImage` component is already handling the proper rendering of image. We don't need to create this manually from our side.

Reply:
Done - use one image instead of two different size